scripts/pages/index.js

```diff
@@ -1,45 +1,31 @@
-    async function getPhotographers() {
-        // Penser à remplacer par les données récupérées dans le json
-        const photographers = [
-            {
-                "name": "Ma data test",
-                "id": 1,
-                "city": "Paris",
-                "country": "France",
-                "tagline": "Ceci est ma data test",
-                "price": 400,
-                "portrait": "account.png"
-            },
-            {
-                "name": "Autre data test",
-                "id": 2,
-                "city": "Londres",
-                "country": "UK",
-                "tagline": "Ceci est ma data test 2",
-                "price": 500,
-                "portrait": "account.png"
-            },
-        ]
-        // et bien retourner le tableau photographers seulement une fois
-        return ({
-            photographers: [...photographers, ...photographers, ...photographers]})
-    }
+// Récupération des données du fichier photographers.json
+async function getPhotographers() {
```

Comment:
`getPhotographers` et `getMedia` sont exactement les mêmes fonctions.. A mettre dans un util ?

[DRY](https://codinglead.co/javascript/what-is-DRY-code)